src/components/MatchCriterion/CommonMatchCriterion.css

```
@@ -5,6 +5,16 @@
     margin-top: 0;
   }
 
+  & [class|="headerInner"] {
+    & > span {
+        color: #b2b2b2;
+    }
+
+    & > div {
+      color: #ffffff;
```

Are there any problems with `var(--bg)` so you prefer to use HEX instead of it?
```suggestion
      color: var(--bg);
```